tests/unit/s2n_security_policies_test.c

```diff
@@ -21,6 +21,7 @@
 int main(int argc, char **argv)
 {
     BEGIN_TEST();
+    EXPECT_SUCCESS(s2n_security_policies_init());
```

Comment:
Something doesn't make me feel good about fix. Are we able to not include the .c file (splitting the tests if necessary)? Another approach is to use a conditional for apple, which doesn't seem to be the most ideal.

Reply:
Even if we split the tests, we would still need to include the .c file to directly access the `security_policy_selection[]`  and the same error would occur in the split test.  I could use a conditional for apple though.